web/ui/react-app/src/pages/targets/EndpointLink.tsx

```diff
@@ -27,7 +27,7 @@ const EndpointLink: FC<EndpointLinkProps> = ({ endpoint, globalUrl }) => {
       {params.length > 0 ? <br /> : null}
       {params.map(([labelName, labelValue]: [string, string]) => {
         return (
-          <Badge color="primary" className={`mr-1 ${labelName}`} key={labelName}>
+          <Badge color="primary" className={`mr-1 ${labelName}`} key={`${labelName}/${labelValue}`}>
```

Comment:
Oh I just noticed that we have a similar bug here (unrelated to your fix) as I fixed elsewhere in https://github.com/prometheus/prometheus/pull/7902/files, where the label name is put into a class name. Will send a fix for that as well.